dpd/p4/headers.p4

```diff
@@ -226,6 +225,40 @@ struct sidecar_ingress_meta_t {
 	// Used for responding to pings
 	mac_addr_t orig_src_mac;	// source mac address before rewriting
 	ipv4_addr_t orig_src_ipv4;	// original ipv4 source
-
 	ipv4_addr_t orig_dst_ipv4;	// original ipv4 target
 }
+
+struct sidecar_egress_meta_t {
```

Comment:
I know that there was metadata here before, but we should probably make an explicit `metadata.p4` and have `headers.p4` just contain actual packet headers.